docs/intro.rst

```diff
@@ -1,5 +1,11 @@
 | |pypi_downloads| |codecov| |windows| |linux| |mac| |bsd|
 
+.. warning::
+   Do not use ``ipdb`` in concert with ``blessed``. It will break ``ipython`` globally.
```

Comment:
This doesn't seem accurate based on what @jquast commented in #91. Rather you should take care about the order of the context managers.

Reply:
I'm also not sure putting this above the introduction makes sense. It seams more like something that would go in a FAQ or Compatibility section.